custom/_legacy/pact/signals.py

```diff
@@ -10,6 +9,7 @@
 
 def process_dots_submission(sender, xform, **kwargs):
     from pact.tasks import recalculate_dots_data, eval_dots_block
+    from pact.utils import get_case_id
```

Comment:
this caused a circular import at some point during development. I don't think it ended up mattering, but probably doesn't hurt to be defensive.